app/database/migrations/20201207174436_make_status_enum.js

```diff
@@ -0,0 +1,19 @@
+
+const status = [
+    'SCHEDULED',
+    'ON_PROCESS',
+    'SUCCESS',
+    'FAIL'
+];
+
+exports.up = function(knex) {
+    return knex.schema.table('jobs', (tbl) => {
+        tbl.string('status');
```

Comment:
This is very good, but I did more research to the syntax and the functionalities of this particular method. It is better to use the native enum method from knex and actually handle it that way. 

        `tbl.enum('status', status, {useNative: true, enumName:'status_enum'}).notNullable().index();`
Also you can delete the .then portion of the code.

Sorry for the confusion